pyscriptjs/src/pyscript.py

```diff
@@ -107,27 +107,6 @@ def format_mime(obj):
 class PyScript:
     loop = loop
 
-    @staticmethod
```

Comment:
I'd not delete it fully but make it use Element(id).write() instead... and add a deprecation warning o warn users that it'll be removed soon.

I'm testing locally and can fix it, just commenting for posterity :)